From c078e177820b6aa1d7032ef0607ad6d59a7551d4 Mon Sep 17 00:00:00 2001
From: Hugues Fruchet <hugues.fruchet@st.com>
Date: Thu, 13 Jun 2019 17:33:30 +0200
Subject: [PATCH 29/31] ARM: dts: stm32: limit OV5640 pixel clock to 77MHz on
 stm32mp157a-av96 board

Limit OV5640 pixel clock to 77MHz in order to fit with DCMI camera interface
maximum bandwidth.

Signed-off-by: Hugues Fruchet <hugues.fruchet@st.com>
Change-Id: I60450e9df85f7f99fe2474e520cfd064baae7fd2
---
 arch/arm/boot/dts/stm32mp157a-av96.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/stm32mp157a-av96.dts b/arch/arm/boot/dts/stm32mp157a-av96.dts
index b6de574..b963385 100644
--- a/arch/arm/boot/dts/stm32mp157a-av96.dts
+++ b/arch/arm/boot/dts/stm32mp157a-av96.dts
@@ -497,6 +497,7 @@
 				remote-endpoint = <&mipid02_0>;
 				clock-lanes = <0>;
 				data-lanes = <1 2>;
+				pclk-max-frequency = <77000000>;
 			};
 		};
 	};
-- 
2.7.4

